classes/services/queryBuilders/UserQueryBuilder.inc.php

```diff
@@ -17,42 +17,4 @@
 
 use Illuminate\Database\Capsule\Manager as Capsule;
 
-class UserQueryBuilder extends \PKP\Services\QueryBuilders\PKPUserQueryBuilder {
-
-	/** @var int Assigned as editor to this section id */
-	protected $assignedToSectionId = null;
-
-	/**
-	 * Limit results to users assigned as editors to this section
-	 *
-	 * @param $sectionId int
-	 *
-	 * @return \PKP\Services\QueryBuilders\UserQueryBuilder
-	 */
-	public function assignedToSection($sectionId) {
-		$this->assignedToSectionId = $sectionId;
-		return $this;
-	}
-
-	/**
-	 * Execute additional actions for app-specific query objects
-	 *
-	 * @param object Query object
-	 * @return object Query object
-	 */
-	public function appGet($q) {
-
-		if (!is_null($this->assignedToSectionId)) {
-			$sectionId = $this->assignedToSectionId;
-
-			$q->leftJoin('section_editors as se', function($table) use ($sectionId) {
-				$table->on('u.user_id', '=', 'se.user_id');
-				$table->on('se.section_id', '=', Capsule::raw((int) $sectionId));
-			});
-
-			$q->whereNotNull('se.section_id');
-		}
-
-		return $q;
-	}
-}
+class UserQueryBuilder extends \PKP\Services\QueryBuilders\PKPUserQueryBuilder { }
```

Comment:
I think you can get rid of this file altogether and change calls to `\App\Services\QueryBuilders\UserQueryBuilder` to `\PKP\Services\QueryBuilders\PKPUserQueryBuilder`.